src/main/java/org/sagebionetworks/web/client/jsinterop/SRC.java

```diff
@@ -11,6 +11,9 @@
 		public static ReactFunctionComponent<EvaluationCardProps> EvaluationCard;
 		public static ReactFunctionComponent<EvaluationEditorPageProps> EvaluationEditorPage;
 		public static ReactFunctionComponent<AccessTokenPageProps> AccessTokenPage;
+		public static ReactFunctionComponent<EmptyProps> DownloadCartPage;
+		public static ReactFunctionComponent<ShowDownloadV2Props> ShowDownloadV2;
+		public static ReactFunctionComponent<DownloadConfirmationProps> DownloadConfirmation;
```

Comment:
3 newly exported SRC components.  The full DownloadCartPage, the notification to put in the Header, and the confirmation component that you seen when you want to add files to the DL.